Makefile

```diff
@@ -53,3 +54,19 @@ release: # run on main branch only
 clean:
 	rm -rf "$(VENV_NAME)"
 	find . -iname "*.pyc" -delete
+
+# Sync to public GitHub repo
+sync:
```

Comment:
Is this meant to be in the public repository?

Reply:
I considered removing the Makefile entirely from the public repository, but opted to keep identical Makefiles in both repos to simplify the contributor setup. While the sync target only functions in the internal repository, I thought is is ok to keep it in public repo to avoid maintaining two separate versions.